experiment/aim.md

```diff
@@ -1 +1,7 @@
-### Aim of the experiment
+Different kinds of calculations require different kinds of input. But some items are to be specified in all the cases. The two most popular computer programs for carrying out ab initio quantum chemical calculations is the "Gaussian" and "Gamess" series.
```

Comment:
'ab initio quantum chemical calculations are the'
in place of
'ab initio quantum chemical calculations is the'